### PR TITLE
chore: Fix #3553

### LINKED ISSUE #3553: Tests / GitHub Actions check why cached download test is really slow
Check why GitHub Action download test is really slow (value on GH Actions: ~4672)

https://github.com/prisma/prisma/blob/adb2116914594bf0d73edcc222e8486201d56dcd/src/packages/fetch-engine/src/__tests__/download.test.ts#L376-L379

### PR DESCRIPTION
**Pull request by timsuchanek**

### CODE REVIEW

File: src/packages/fetch-engine/src/getHash.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import crypto from 'crypto'
+import fs from 'fs'
+
+export function getHash(filePath: string): Promise<string> {
```
Comment (Jolg42) [inline src/packages/fetch-engine/src/getHash.ts:4]: Could we use this here too https://github.com/prisma/prisma/blob/284891a1d04a1d98f358b5cd141c79c0106ed0c3/src/packages/fetch-engine/src/downloadZip.ts#L84

And then remove the hasha dependency?

File: src/packages/fetch-engine/src/__tests__/download.test.ts [hunk 2]
```diff
@@ -327,14 +327,16 @@ describe('download', () => {
         'rhel-openssl-1.1.x',
         'windows',
         'linux-musl',
+        'freebsd12',
```
Comment (Jolg42) [inline src/packages/fetch-engine/src/__tests__/download.test.ts:330]: We do not provide binaries for freebsd12, why did you add it? 🤔

File: src/packages/fetch-engine/src/getHash.ts (hunk 1 above)
Comment (timsuchanek) [reply src/packages/fetch-engine/src/getHash.ts:4]: We still use hasha in `downloadZip`.

File: src/packages/fetch-engine/src/__tests__/download.test.ts (hunk 2 above)
Comment (timsuchanek) [reply src/packages/fetch-engine/src/__tests__/download.test.ts:330]: To make sure our behavior there is consistent.
But as for this test it can indeed be removed

File: src/packages/fetch-engine/src/getHash.ts (hunk 1 above)
Comment (Jolg42) [reply src/packages/fetch-engine/src/getHash.ts:4]: Yes this is what I pointed out in my comment.
Could we use `getHash` everywhere instead?
